ksql-engine/src/test/java/io/confluent/ksql/KsqlEngineTest.java

```diff
@@ -136,10 +136,15 @@ public void closeEngine() {
 
   @Test
   public void shouldCreatePersistentQueries() {
+    // Given:
+    topicClient.preconditionTopicExists("test2", 3, (short) 1, Collections.emptyMap());
```

Comment:
All these `preconditionTopicExists` calls for the source topics should be done in setUp.